### PR TITLE
Add data on whether we are in gitpod. Use gitpod user id hash for id

### PR DESCRIPTION
**Pull request by janedegtiareva**

### CODE REVIEW

**Comment by janedegtiareva** [comment]
Fix for https://app.zenhub.com/workspace/o/near/devx/issues/194

**Comment by janedegtiareva** [comment]
Fix for https://github.com/near/devx/issues/194

**Comment by janedegtiareva** [comment]
Tested on windows local + gitpod.
*gitpod (see screenshot)
*local with missing consent data in settings (question hasn't been asked)
*local with user opt out of tracking
*local with user opt in to tracking 
Screenshot (29)

File: utils/eventtracking.js [hunk 1]
```diff
@@ -3,26 +3,44 @@ const mixpanel = require('mixpanel').init(MIXPANEL_TOKEN);
...
 
+const isGitPod = () => {
+    return !!process.env.GITPOD_WORKSPACE_URL;
+};
+
+const getGitPodUserHash = () => {
+    if (!process.env.GITPOD_GIT_USER_NAME) {
+        return null;
+    }
+    return crypto.createHash('sha256').update(process.env.GITPOD_GIT_USER_NAME, 'utf8').digest('hex').toString();
```
Comment (vgrichina) [inline utils/eventtracking.js:21]: Cool that we can track sessions across users in anonymized way. Wonder if it's better to use `GITPOD_GIT_USER_EMAIL` as my understanding is that `GITPOD_GIT_USER_NAME` is full name (so not necessarily unique).

Comment (janedegtiareva) [reply utils/eventtracking.js]: sure, updated.

**Pull request merged**